From ecf04870c349f8f7ac889ec50d465a2d1dec517c Mon Sep 17 00:00:00 2001
From: Bastien Nocera <hadess@hadess.net>
Date: Fri, 28 Feb 2020 17:18:27 +0100
Subject: [PATCH] plparser: Fix header typo

plparse/totem-pl-parser.c:1308: Warning: TotemPlParser: totem_pl_parser_save_finish: unknown parameter 'async_result' in documentation comment, should be 'result'
---
 plparse/totem-pl-parser.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/plparse/totem-pl-parser.h b/plparse/totem-pl-parser.h
index fc1397b..6205aa6 100644
--- plparse/totem-pl-parser.h
+++ plparse/totem-pl-parser.h
@@ -358,7 +358,7 @@ void totem_pl_parser_save_async (TotemPlParser        *parser,
 				 GAsyncReadyCallback   callback,
 				 gpointer              user_data);
 gboolean totem_pl_parser_save_finish (TotemPlParser   *parser,
-				      GAsyncResult    *result,
+				      GAsyncResult    *async_result,
 				      GError         **error);
 
 void	   totem_pl_parser_add_ignored_scheme (TotemPlParser *parser,
-- 
GitLab

